### PR TITLE
app-misc/elasticsearch: bump/cleanup

### PR DESCRIPTION
**Pull request by hydrapolic**
- bump to 2.4.3
- simplify 5.1.1 (we are not using MY_PN)
- drop unsupported versions

for version 2.4.3: https://bugs.gentoo.org/show_bug.cgi?id=590372

### CODE REVIEW

**Comment by gktrk** [comment]
How will the "improve ebuild" commit affect the current installations? Do we need a revbump for this?

File: app-misc/elasticsearch/elasticsearch-2.4.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	doins config/*
+	rm -rf config || die
+
+	insinto /usr/share/${PN}
+	doins -r ./*
+
+	exeinto /usr/share/${PN}/bin
+	doexe "${FILESDIR}/elasticsearch-systemd-pre-exec"
+
+	chmod +x "${D}"/usr/share/${PN}/bin/*
```
Comment (gktrk) [inline app-misc/elasticsearch/elasticsearch-2.4.3.ebuild]: Missing `|| die`

**Comment by hydrapolic** [comment]
Hello,the improve ebuild is just a simplification because it seems like this ebuild started as elasticsearch-bin and made it into gentoo as elasticsearch and thus there is this unneded transformation MY_PN="${PN%-bin}". From the user perspective, there is no change at all, so I suppose we do not need a revbump.

**Comment by gktrk** [comment]
Sorry I missed that it's a cosmetic change.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/e2e223f...2e4f510. Thanks!

**Comment by hydrapolic** [comment]
Thank you.

**Pull request closed without merging**